oteapi_dlite/strategies/oceanlab_influx_parser.py

```diff
@@ -0,0 +1,296 @@
+        for _ in [
+            {
+                "measurement": "ctd_conductivity_munkholmen",
+                "field": "conductivity",
+            },
+            {
+                "measurement": "ctd_density_munkholmen",
+                "field": "density",
+            },
+            {
+                "measurement": "ctd_salinity_munkholmen",
+                "field": "salinity",
+            },
+            {
+                "measurement": "ctd_pressure_munkholmen",
+                "field": "pressure",
+            },
+        ]
+    ]
+
+
+class InfluxParseStrategyConfig(ParserConfig):
+    """DLite excel parse strategy  config."""
+
+    configuration: Annotated[
+        InfluxParseParseConfig,
+        Field(
+            description="DLite InfluxDB parse strategy-specific configuration."
+        ),
+    ]
+
+
+class InfluxParserUpdate(DLiteSessionUpdate):
+    """Class for returning values from DLite json parser."""
+
+    inst_uuid: Annotated[
+        str,
+        Field(
+            description="UUID of new instance.",
+        ),
+    ]
+    label: Annotated[
+        str,
+        Field(
+            description="Label of the new instance in the collection.",
+        ),
+    ]
+
+
+@dataclass
+class InfluxParseStrategy:
+    """Parse strategy for Influx DB.
+
+    **Registers strategies**:
+
+    - `("parserType",
+        "influx/vnd.dlite-influx")`
+
+    """
+
+    parse_config: InfluxParseStrategyConfig
+
+    def initialize(self) -> DLiteSessionUpdate:
+        """Initialize."""
+        collection_id = (
+            self.parse_config.configuration.collection_id
+            or get_collection().uuid
+        )
+        return DLiteSessionUpdate(collection_id=collection_id)
+
+    def get(self) -> InfluxParserUpdate:
+        """Execute the strategy.
+
+        This method will be called through the strategy-specific endpoint
+        of the OTE-API Services.
+
+        Returns:
+            DLite instance.
+
+        """
+        config = self.parse_config.configuration
+        try:
+            # Update dlite storage paths if provided
+            if config.storage_path:
+                for storage_path in config.storage_path.split("|"):
+                    dlite.storage_path.append(storage_path)
+        except Exception as e:
+            logger.error(f"Error during update of DLite storage path: {e}")
+            raise RuntimeError("Failed to update DLite storage path.") from e
+
+        try:
+            env = jinja2.Environment(loader=jinja2.BaseLoader, autoescape=True)
+            env.globals.update(enumerate=enumerate, str=str)
+            bucket = f"{config.DATABASE}/{config.RETPOLICY}"
+            configuration = {
+                "bucket": bucket,
+                "timeRange": config.time_range,
+                "limitSize": str(config.size_limit),
+                "measurements": [
+                    measurement.dict() for measurement in config.measurements
+                ],
```

Comment:
`dict()` is deprecated.
This should work - if it doesn't add the `mode="json"` parameter to the `model_dump` method :)
```suggestion
                "measurements": config.measurements.model_dump(),
```